Add HDFS in kubernetes.md (#889)
risingwavelabs/risingwave-docs: processing/maintain-wide-table-with-table-sinks.mdx

ON CONFLICT DO UPDATE IF NOT NULL;

CREATE SINK sink1 INTO wide_d (v1, k) AS
  SELECT v1, k FROM d1;
CREATE SINK sink2 INTO wide_d (v2, k) AS
  SELECT v2, k FROM d2;
CREATE SINK sink3 INTO wide_d (v3,k) AS
  SELECT v3, k FROM d3;
```

<Note>

> `ON CONFLICT` only controls how INSERT and UPDATE events are handled. By default, `CREATE SINK ... INTO` forwards DELETE events as well, so if any upstream deletes a row (same primary key), the corresponding row in the wide table will be deleted.

Reviewer: ```suggestion
`ON CONFLICT` only controls how `INSERT` and `UPDATE` events are handled. By default, `CREATE SINK ... INTO` forwards `DELETE` events as well. If any upstream deletes a row with a given primary key, the corresponding row in the wide table will be deleted.
```

> If you don't want DELETE events from a sink to remove rows in the wide table, force the sink to be append-only:

Reviewer: ```suggestion
To prevent `DELETE` events from removing rows in the wide table, force the sink to be append-only:
```
